jwst/ami/matrix_dft.py

```diff
@@ -1,65 +1,67 @@
-"""
-    Summary
-    -------
-    MatrixDFT: Matrix-based discrete Fourier transforms for computing PSFs.
-    See Soummer et al. 2007 JOSA
-    The main user interface in this module is a class MatrixFourierTransform.
-    Internally this will call one of several subfunctions depending on the
-    specified centering type. These have to do with where the (0, 0) element of
-    the Fourier transform is located, i.e. where the PSF center ends up.
-        - 'FFTSTYLE' centered on one pixel
-        - 'SYMMETRIC' centerd on crosshairs between middle pixel
-        - 'ADJUSTABLE', always centered in output array depending on
-          whether it is even or odd
-    'ADJUSTABLE' is the default.
-    This module was originally called "Slow Fourier Transform", and this
-    terminology still appears in some places in the code.  Note that this is
-    'slow' only in the sense that if you perform the exact same calculation as
-    an FFT, the FFT algorithm is much faster. However this algorithm gives you
-    much more flexibility in choosing array sizes and sampling, and often lets
-    you replace "fast calculations on very large arrays" with "relatively slow
-    calculations on much smaller ones".
-
-    Example
-    -------
-    mf = matrixDFT.MatrixFourierTransform()
-    result = mf.perform(pupilArray, focalplane_size, focalplane_npix)
+"""Matrix-based discrete Fourier transforms for computing PSFs.
+
+Summary
+-------
+MatrixDFT: Matrix-based discrete Fourier transforms for computing PSFs.
+See Soummer et al. 2007 JOSA
```

Comment:
Is the URL to this paper in the docs?  It not, maybe put it here.

Reply:
Can do